initfiles/componentfiles/configxml/esp.xsd.in

```diff
@@ -999,6 +999,13 @@
                     </xs:appinfo>
                 </xs:annotation>
             </xs:attribute>
+            <xs:attribute name="txSummaryStyle" type="xs:text" use="optional" default="text">
```

Comment:
No corresponding attribute for txSummaryGroup?

Reply:
Thanks added.